exams/pearson.py

```diff
@@ -23,6 +23,7 @@
     delimiter='\t',
 )
 
+PEARSON_DATE_FORMAT = "%Y/%m/%d"
```

Comment:
I had the same thought, but have done it in a separate branch that includes a refactor of all the pearson code into its own module: https://github.com/mitodl/micromasters/compare/nl/2076-exam-auth-exports...nl/refactor-pearson-to-module

I'm happy to include those changes in this PR, but I had figured a separate PR just to refactor was easier on reviewers.